components/higher-order/with-focus-outside/test/index.js

```diff
@@ -12,9 +12,12 @@ import { Component } from '@wordpress/element';
  * Internal dependencies
  */
 import withFocusOutside from '../';
+import ReactDOM from 'react-dom';
 
 jest.useFakeTimers();
 
+let wrapper, callback;
```

Comment:
Can we rename `callback` to `onFocusOutside` to make it easier to read since it is now hidden inside `beforeEach` method?

Reply:
change implemented